app/models/course.rb

```diff
@@ -263,10 +263,22 @@ def sites_not_associated_with_course
   end
 
   def has_bursary?
+    # If course name contains "with"
+    return false if name.include?("with")
```

Comment:
Primary with mathematics has a bursary though:
https://getintoteaching.education.gov.uk/funding-my-teacher-training/bursaries-and-scholarships-for-teacher-training

Reply:
Live example showing bursary: https://find-postgraduate-teacher-training.education.gov.uk/course/2AT/2T89#section-financial-support